charts/fleet-agent/templates/deployment.yaml

```diff
@@ -59,6 +59,9 @@ spec:
             drop:
             - ALL
         {{- end }}
+        volumeMounts:
```

Comment:
Why do we need this?

Reply:
To get rid of:
`fleet-agent-0 fleet-agent I0705 14:38:47.514247       1 cached_discovery.go:93] failed to write cache to /.kube/cache/discovery/10.43.0.1_443/batch/v1/serverresources.json due to mkdir /.kube: permission denied?`